src/pages/EditMonitor.vue

```diff
@@ -156,16 +156,27 @@
                             </div>
 
                             <!-- Keyword -->
-                            <div v-if="monitor.type === 'keyword' || monitor.type === 'grpc-keyword'" class="my-3">
+                            <div v-if="monitor.type === 'keyword' || monitor.type === 'grpc-keyword' || monitor.type === 'real-browser'" class="my-3">
                                 <label for="keyword" class="form-label">{{ $t("Keyword") }}</label>
-                                <input id="keyword" v-model="monitor.keyword" type="text" class="form-control" required>
+                                <input
+                                    id="keyword"
+                                    v-model="monitor.keyword"
+                                    type="text"
+                                    class="form-control"
+                                    :required="monitor.type === 'keyword' || monitor.type === 'grpc-keyword'"
+                                >
                                 <div class="form-text">
-                                    {{ $t("keywordDescription") }}
+                                    <span v-if="monitor.type === 'real-browser'">
+                                        {{ $t("keywordDescription") }} {{ $t("Optional") }}.
+                                    </span>
+                                    <span v-else>
+                                        {{ $t("keywordDescription") }}
+                                    </span>
```

Comment:
Let's not add a second way of doing this.
For noting which field are required/ optional, we really should have a slightly different solution, but this is out of scope for this pr


```suggestion
                                    <span>
                                        {{ $t("keywordDescription") }}
                                    </span>
```